tag.py

```diff
@@ -28,23 +28,38 @@ def run_command(args):
     print(f'--- Running command: {" ".join(args)}')
     subprocess.check_call(args)
 
-def commit_change(version):
+def commit_change(version, force):
     run_command(['git', 'add', 'roc/version.go'])
-    run_command(['git', 'commit', '-m', f'Release {version}'])
+    try:
+        run_command(['git', 'commit', '-m', f'Release {version}'])
+    except subprocess.CalledProcessError:
```

Comment:
I think it's better to check if we have something to commit instead of catching a general error here

Reply:
Done! It feels better this way, thank you.

I did the same tests as in the previous privatebin.

Reply:
Small suggestion: we could first run "git add", and only then "git diff", but with "--cached" option. This way we will handle all 3 situations: when roc/version.go is unchanged, changed and not staged, or changed and staged.